tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+minversion = 3.2.0
+envlist = py36,py38,py39,pylint,coverage
+ignore_basepython_conflict = true
+
+[testenv]
+basepython = python3
+usedevelop = True
+install_command = pip install {opts} {packages}
+deps = -r{toxinidir}/test-requirements.txt
+commands = nosetests
+
+[testenv:venv]
+commands = {posargs}
+
+[testenv:pylint]
```

Comment:
`pylint` should be added to `test-requirements.txt` else it will fail.

```shell
$ tox -r -e pylint 
pylint create: /home/hberaud/dev/perso/podman-py/.tox/pylint
pylint installdeps: -r/home/hberaud/dev/perso/podman-py/test-requirements.txt
pylint develop-inst: /home/hberaud/dev/perso/podman-py
pylint installed: coverage==5.3,extras==1.0.0,fixtures==3.0.0,linecache2==1.0.0,nose==1.3.7,pbr==5.5.1,-e git+git@github.com:4383/podman-py@842e96dd879564abb062e9b70095923cf080d133#egg=podman,python-mimeparse==1.6
.0,six==1.15.0,testtools==2.4.0,traceback2==1.4.0,unittest2==1.1.0
pylint run-test-pre: PYTHONHASHSEED='2428089515'
pylint run-test: commands[0] | pylint podman
ERROR: InvocationError for command could not find executable pylint
```